core/src/main/scala/org/bykn/bosatsu/rankn/DefinedType.scala

```diff
@@ -23,15 +23,26 @@ case class DefinedType[+A](
    * A type with exactly one constructor is a struct
    */
   def isStruct: Boolean = constructors.lengthCompare(1) == 0
+
+  /**
+   * A newtype is just a wrapper for another type.
+   * It could be removed statically from the program
+   */
+  def isNewType: Boolean =
```

Comment:
Could this be a `val` (or `lazy val`) instead of `def`?

Reply:
It could be. Might as well. Val is probably better.